.github/workflows/main.yml

```diff
@@ -247,7 +247,7 @@ jobs:
 
     - name: Build h5py from source
       run: |
-        git clone https://github.com/h5py/h5py.git h5py && cd h5py && git pull
+        git clone https://github.com/h5py/h5py.git h5py && cd h5py && git checkout 2.9.0
```

Comment:
We should also freeze the version of h5py in the Dockerfile. Otherwise it will fail next time that step is not cached.

Reply:
Good catch, I've just done that.